autopush-common/src/db/bigtable/bigtable_client/mod.rs

```diff
@@ -141,6 +141,7 @@ impl BigTableClientImpl {
         &self,
         row_key: &str,
         timestamp_filter: Option<u64>,
+        table_name: Option<String>,
```

Comment:
Why the new optional argument? I don't see it being used (`None` is always passed)